test/var_sets_test.py

```diff
@@ -24,6 +25,24 @@ def tearDown(self):
         self._api.workspaces.destroy(
             workspace_name=self._ws["data"]["attributes"]["name"])
 
+    def create_multiple_varsets(self, var_set_name, create_payload):
+        varset_ids = list()
+        for i in range(TFC_PAGE_SIZE):
+            create_payload["data"]["attributes"]["name"] = f"{var_set_name}{i}"
+            varset_ids.append(self._api.var_sets.create(create_payload)["data"]["id"])
+
+        return varset_ids
+
+    def create_multiple_varset_vars(self, varset_id, add_var_to_varset_payload):
+        variable_ids = list()
+        for i in range(TFC_PAGE_SIZE):
+            add_var_to_varset_payload["data"]["attributes"]["key"] = f"{self._variable_test_key}{i}"
+            add_var_to_varset_payload["data"]["attributes"]["value"] = f"{self._variable_test_value}{i}"
+
+            variable_ids.append(self._api.var_sets.add_var_to_varset(varset_id, add_var_to_varset_payload)["data"]["id"])
+
+        return variable_ids
+
```

Comment:
Either do this inside of the existing function, or put an underscore in front of the function name (_create_multiple_varsets). Doesn't functionally work any differently, but matches the structure of the existing tests.

Reply:
I moved these to the existing function since each of my functions is only called once, and I didn't need a worry about modifications to the payloads since they don't get re-used. I also changed some of the formatting as I realized I had neglected to run pylint against the test file; several of my lines were too long.